cmd/pebble/cmd_help.go

```diff
@@ -167,16 +167,20 @@ type helpCategory struct {
 // helpCategories helps us by grouping commands
 var helpCategories = []helpCategory{{
 	Label:       "Basics",
-	Description: "basic management",
-	Commands:    []string{"start", "stop", "autostart"},
+	Description: "basic service management",
+	Commands:    []string{"autostart", "run", "services", "start", "stop"},
```

Comment:
Some of this CLI help stuff had gotten out of date.

Reply:
We need to either remove "run" from the list, or remove "service" from the description. If we keep it in the list, we need to reorder it first as well, as that's how to get Pebble running at all.

Reply:
I removed "service" from the description and moved "run" to the first item (I had organized them alphabetically, but that's probably not the most useful way in a short list like this.